src/templates/preview.hbs

```diff
@@ -0,0 +1,129 @@
+        overflow: auto;
+      }
+
+      #message_headers dl {
+        float: left;
+        line-height: 1.3em;
+        padding: 0;
+      }
+
+      #message_headers dt {
+        width: 92px;
+        margin: 0;
+        float: left;
+        text-align: right;
+        font-weight: bold;
+        color: #7f7f7f;
+      }
+
+      #message_headers dd {
+        margin: 0 0 0 102px;
+      }
+
+      h2 {
+        font-family: "Segoe UI", "Helvetica Neue", Arial, sans-serif;
+        font-size: 14px;
+        color: darkslategrey;
+        background-color: #fcfcfc;
+        border-bottom: 1px solid #dedede;
+        border-top: 1px solid #dedede;
+
+        margin: 0 0 16px;
+        padding: 8px 16px;
+      }
+
+      pre#message_body {
+        padding: 8px 16px;
+        white-space: pre-wrap;
+      }
+
+      iframe {
+        border: 0;
+        width: 100%;
+        height: 100%;
+        margin: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="container">
+      <div id="message_headers">
+        <dl>
+          <dt>From:</dt>
+          <dd>{{from}}</dd>
+
+          {{#if sender}}
+            <dt>Sender:</dt>
+            <dd>{{sender}}</dd>
+          {{/if}}
+
+          {{#if replyTo}}
+            <dt>Reply-To:</dt>
+            <dd>{{replyTo}}</dd>
+          {{/if}}
+
+          {{#if subject}}
+            <dt>Subject:</dt>
+            <dd><strong>{{subject}}</strong></dd>
+          {{/if}}
+
+          {{#if date}}
+            <dt>Date:</dt>
+            <dd>{{date}}</dd>
+          {{/if}}
+
+          {{#if to}}
+            <dt>To:</dt>
+            <dd>{{to}}</dd>
+          {{/if}}
+
+          {{#if cc}}
+            <dt>CC:</dt>
+            <dd>{{cc}}</dd>
+          {{/if}}
+
+          {{#if bcc}}
+            <dt>BCC:</dt>
+            <dd>{{bcc}}</dd>
+          {{/if}}
+        </dl>
+      </div>
+
+      {{#if text}}
+        <h2>Text version</h2>
+        <pre id="message_body">{{text}}</pre>
+      {{/if}}
+
+      {{#if html}}
+        <h2>HTML version</h2>
+        <iframe seamless="seamless" srcdoc="<base target='_top'>{{html}}"></iframe>
+      {{/if}}
+    </div>
+
+    <script type="text/javascript">
+      /*! srcdoc-polyfill - v0.1.1 - 2013-03-01
+      * http://github.com/jugglinmike/srcdoc-polyfill/
+      * Copyright (c) 2013 Mike Pennisi; Licensed MIT */
+      (function(t,e){var c,n,o=t.srcDoc,r=!!("srcdoc"in e.createElement("iframe")),i={compliant:function(t,e){e&&t.setAttribute("srcdoc",e)},legacy:function(t,e){var c;t&&t.getAttribute&&(e?t.setAttribute("srcdoc",e):e=t.getAttribute("srcdoc"),e&&(c="javascript: window.frameElement.getAttribute('srcdoc');",t.setAttribute("src",c),t.contentWindow&&(t.contentWindow.location=c)))}},s=t.srcDoc={set:i.compliant,noConflict:function(){return t.srcDoc=o,s}};if(!r)for(s.set=i.legacy,n=e.getElementsByTagName("iframe"),c=n.length;c--;)s.set(n[c])})(this,this.document);
```

Comment:
We should not have minified code, this is not easy to review, we could just return the html of the template directly we don't need iframe or others metadata